src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -489,7 +474,7 @@ private VariantContext filterSiteByAlleleSpecificVQSLOD(VariantContext mergedVC,
         builder.attribute(GATKVCFConstants.AS_VQS_LOD_KEY, relevantVqsLodMap.values().stream().map(val -> val.equals(Double.NaN) ? VCFConstants.EMPTY_INFO_FIELD : val.toString()).collect(Collectors.toList()));
         builder.attribute(GATKVCFConstants.AS_YNG_STATUS_KEY, new ArrayList<>(relevantYngMap.values()));
 
-        if (!onlyAnnotate) {
+        if (filteringType.equals(ExtractCohort.FilteringType.SITES)) {
```

Comment:
where is this finding the variable filteringType? the parameter passed in is onlyAnnotate.

Reply:
ooof--nice catch, thank you!